easybuild/tools/github.py

```diff
@@ -264,7 +264,10 @@ def fetch_latest_commit_sha(repo, account, branch='master', github_user=None, to
             break
 
     if res is None:
-        raise EasyBuildError("No branch with name %s found in repo %s/%s (%s)", branch, account, repo, data)
+        error_msg = "No branch with name %s found in repo %s/%s" % (branch, account, repo)
+        if len(data) >= GITHUB_MAX_PER_PAGE:
```

Comment:
bleh, it's far from trivial to do this, since the header info isn't readily available in the result of `github_api_get_request` (which is used by `fetch_latest_commit_sha`)...

Is it worth the trouble? This is only about a slightly more informative error message...